subprojects/docs/src/docs/release/notes.md

```diff
@@ -6,9 +6,11 @@ This will result in significantly reduced Java compilation time in subsequent bu
 Chances are caches in those `.gradle/` directories have accumulated a few (or a few dozen) gigabytes over time.
 If so, you'll be relieved to know that Gradle will now [periodically clean up unused `/caches`](#periodic-cache-cleanup) under `GRADLE_USER_HOME` and project root directories. 
 
-A moment you have anticipated is nearly here, as the [Kotlin DSL reaches version 1.0 RC1](https://github.com/gradle/kotlin-dsl/releases/tag/v1.0-RC1) with this release of Gradle.
-Configuration avoidance, script compilation caching, buildSrc refactoring propagation, and lots of DSL polish make this the release to try.
-_If you are interested in using the Kotlin DSL, please check out the [migration guide](https://guides.gradle.org/migrating-build-logic-from-groovy-to-kotlin/) and file issues in the [gradle/kotlin-dsl](https://github.com/gradle/kotlin-dsl) project._ 
+A moment you have anticipated is nearly here, as the [Kotlin DSL reaches version 1.0 RC1](https://github.com/gradle/kotlin-dsl/releases/tag/v1.0-RC1). 
+Configuration avoidance, `buildSrc` refactoring propagation to the IDE, and lots of DSL polish make this the release to try.
+The RC phase will span until the next Gradle version that will ship with the Gradle Kotlin DSL 1.0.
```

Comment:
Should we state this plainly "Gradle Kotlin DSL 1.0 will ship with the next version of Gradle"?